packages/ripple-binary-codec/package.json

```diff
@@ -38,6 +38,7 @@
   "readmeFilename": "README.md",
   "prettier": "@xrplf/prettier-config",
   "engines": {
-    "node": ">=10.22.0"
+    "node": ">=14",
+    "npm": ">=7.0.0 < 8.0.0"
```

Comment:
The npm restrictions here mean that people who are installing the package also need to abide by them. This will cause problems (and has in the past - some legacy systems are still using npm v6).

Reply:
The same is true with the node ones. Even if we don't run tests on lower than 14, I don't see why we should prevent people from continuing to run legacy systems with our code.